packages/wrangler/src/dev/miniflare.ts

```diff
@@ -655,23 +743,25 @@ export function buildMiniflareBindingOptions(config: MiniflareBindingsConfig): {
 
 	const wrappedBindings: WorkerOptions["wrappedBindings"] = {};
 	if (bindings.ai?.binding) {
-		externalWorkers.push({
-			name: `${EXTERNAL_AI_WORKER_NAME}:${config.name}`,
-			modules: [
-				{
-					type: "ESModule",
-					path: "index.mjs",
-					contents: EXTERNAL_AI_WORKER_SCRIPT,
+		if (!getFlag("MIXED_MODE")) {
+			externalWorkers.push({
+				name: `${EXTERNAL_AI_WORKER_NAME}:${config.name}`,
+				modules: [
+					{
+						type: "ESModule",
+						path: "index.mjs",
+						contents: EXTERNAL_AI_WORKER_SCRIPT,
+					},
+				],
+				serviceBindings: {
+					FETCHER: AIFetcher,
 				},
-			],
-			serviceBindings: {
-				FETCHER: AIFetcher,
-			},
-		});
+			});
 
-		wrappedBindings[bindings.ai.binding] = {
-			scriptName: `${EXTERNAL_AI_WORKER_NAME}:${config.name}`,
-		};
+			wrappedBindings[bindings.ai.binding] = {
+				scriptName: `${EXTERNAL_AI_WORKER_NAME}:${config.name}`,
+			};
+		}
```

Comment:
This PR doesn't affect AI, I don't think, so probably shouldn't remove this?

Reply:
This PR also addresses AI... however maybe it'd be better to revert the AI changes and deal with them later 🤔

Reply:
AI added to the changelog 😄